packages/terra-menu/CHANGELOG.md

```diff
@@ -2,6 +2,15 @@
 
 ## Unreleased
 
+* Added
+  * Allow screen readers to recognise as menu button
+  * Updated menu button to use down caret for visual context
+  * Added context for menu item index, item with submenu and selected/unselected state
+
+* Changed
+  * Updated WDIO test and screenshots
+  * Updated jest snapshots
+
```

Comment:
Since CHANGELOGs are consumer facing, we don't need to add entries for dev changes.

Reply:
Updated in https://github.com/cerner/terra-framework/pull/1626/commits/131a931bb445c9aeab73a4e5174592ed7d503b99